app/views/landing/index.html.haml

```diff
@@ -1,7 +1,4 @@
 .container
-  .row.mt-4
-    .col 
-      %h1.text-center David's Blog
   .row
     .col 
```

Comment:
Nit: I don't think you really need a `row` here given that there is just a single unresponsive `col` in it.  You should be able to get rid of both `row` and `col`